bin/exporter

```diff
@@ -28,4 +28,5 @@ AuroraBootstrapper::Exporter.new( client: client,
                            export_bucket: bukkit,
                       blacklisted_tables: blacklisted_tables,
                       whitelisted_tables: whitelisted_tables,
-                      blacklisted_fields: blacklisted_fields ).export!
\ No newline at end of file
+                      blacklisted_fields: blacklisted_fields,
+                      notifier: Notifier.new ).export!
```

Comment:
looks like this gets overridden in the `Exporter` initializer. Can we remove this?